test/PhpSessionPersistenceTest.php

```diff
@@ -188,8 +188,9 @@ public function testPersistSessionGeneratesCookieWithNewSessionIdIfSessionWasReg
      */
     public function testPersistSessionReturnsResponseWithSetCookieHeaderIfSessionCookiePresent()
     {
-        $request = $this->createSessionCookieRequest('use-this-id');
+        $request = $this->createSessionCookieRequest();
```

Comment:
Why did you change this line? This test was originally partially validating that a new session identifier was created when persisting a session; the only change to this particular test should be the change you introduce on line 193.

Reply:
I reverted that change and instead used `time()` to make sure that the session is indeed considered changed and a cookie is set.

If we don't do that, then the test will succeed on first run, but fail on any subsequent runs, because session saved on disk will be picked up and considered un-changed and thus a cookie will not be set, failing the test.

I believe a more proper solution is to always run all tests in a controlled environnement. `testNoMultipleEmptySessionFilesAreCreatedIfNoSessionCookiePresent` is one of a few tests that setup a custom `save_path` and take care of cleaning up that folder before running tests. I think this concept should be generalized to all tests and be done in `setUp()` instead of being copy/pasted as it is now. But that seemed out of scope, hence my quick fix. But if you agree on the principle I'd gladly submit a second PR for that.

Reply:
:+1: to both: the rationale for the change, and an additional PR to move session cleaning to `setUp()`.

I'll get this merged soon!